zipkin-example/src/main/java/com/example/helloworld/HelloWorldApplication.java

```diff
@@ -43,6 +45,9 @@ public void run(HelloWorldConfiguration configuration,
         final Client client = new ZipkinClientBuilder(environment, brave)
                 .build(configuration.getZipkinClient());
 
```

Comment:
Hi guys. Sorry for the delay. I've been trying to add test cases but am running into issues. Perhaps I'm going a bit overkill. My effort so far has been to write a DropwizardAppRule type integreatino test (mentioned at the bottom of this page http://www.dropwizard.io/0.8.0-rc1/docs/manual/testing.html) which will bootstrap the zipkin-example app and call one of it's endpoints. Once I finally get that working I'll then add some RxJava code to the test endpoint to make > 1 call and then somehow drill into the Brave/SpanCollector to assert that the traceID is identical for each request. I've got DropwizardAppRule working fine in my app but when I copy and adapt it into zipkin-example I get missing class issues in my IDE and the error below in the command line. Perhaps you guys who are more familiar with this code base and/or Brave would be able to knock up a test faster.